app/services/SeedingService.php

```diff
@@ -27,13 +27,13 @@ static function createTestUser($fn, $ln)
         $newUser->gender = Gender::M;
         $newUser->birthdate = date_create("1996-01-01");
         $newUser->status = UserStatus::ACTIVE;
-        em()->persist($newUser);
-        em()->flush();
+        $em->persist($newUser);
```

Comment:
Extracting the `em` is also what is called "dependency injection".

In general, it's a better pattern. We should try to do that with all our services